README.md

```diff
@@ -1044,19 +1058,25 @@ The user must verify the factor-specific challenge.
 #### [verify(options)](http://developer.okta.com/docs/api/resources/authn.html#verify-factor)
 
   - `passCode` - OTP sent to device
+  - `autoPush` - Optional parameter to send a push notification immediately the next time `verify` is called on a push factor
 
 ```javascript
 transaction.verify({
-  passCode: '615243'
+  passCode: '615243',
+  autoPush: true
 });
 ```
 
-#### [poll()](http://developer.okta.com/docs/api/resources/authn.html#activate-push-factor)
+#### [poll(options)](http://developer.okta.com/docs/api/resources/authn.html#activate-push-factor)
 
-Poll until factorResult is not WAITING. Throws AuthPollStopError if prev or cancel is called.
+  - `autoPush` - Send a push notification immediately the next time `verify` is called on a push factor
```

Comment:
Sweet. Change this to `Optional parameter to send a push notification immediately the next time `verify` is called on a push factor` to match the others and 🚢